tests/providers/google/marketing_platform/operators/test_campaign_manager_system.py

```diff
@@ -17,8 +17,8 @@
 # under the License.
 import pytest
```

Comment:
In the new design this file is not required. Please ensure that the setup/teardown fixtures are migrated to the DAG itself.